numba/tests/test_random.py

```diff
@@ -394,7 +394,6 @@ def _check_randrange(self, func1, func2, func3, ptr, max_width, is_numpy, tp=Non
             ints.append(func2(5, 500000000))
             if func3 is not None:
                 ints.append(func3(5, 500000000, 3))
-        self.assertEqual(len(ints), len(set(ints)), ints)
```

Comment:
I'm not surprised that this is unstable as it seems like the seed isn't set anywhere, agree with removing it.